### ISSUE TITLE
Bundled I18N::LangTags::Detect fails to load

### ISSUE THREAD

**Issue by thoukydides** [LMS-Community/slimserver#715]
With Perl v5.30.0 on Ubuntu 20.04.3 LTS, a plugin that attempts to `use I18N::LangTags::Detect;` results in the error:
```
Slim::bootstrap::tryModuleLoad (286) Warning: Module [Plugins::XXX::Plugin] failed to load:
Can't use 'defined(%hash)' (Maybe you should just omit the defined()?) at /usr/share/squeezeboxserver/CPAN/I18N/LangTags/Detect.pm line 140.
```

This was fixed by https://github.com/Perl/perl5/commit/08f2910fedd8fc4c335b753d655fd4e769da3784 in verion 1.04 of `I18N::LangTags::Detect`, but Logitech Media Server is still shipping with version 1.03. (The current version is 1.08.)

**Comment by michaelherger** [comment]
Thanks for the hint! This module has been part of Perl (core) since v5.12. Time to get rid of our outdated copy.

Which plugin would be using it? Can you confirm it's working still (new LMS 8.3 builds will be out soon)?

**Comment by thoukydides** [comment]
I have just confirmed that it is fixed in the `logitechmediaserver_8.3.0~1640066892_all.deb` nightly build. Nothing else is obviously broken. Thank you.

I noticed the problem with an unpublished plugin that I developed to integrate with some of my home automation systems. It uses the `Number::Phone` package to format and lookup Calling Line Identification Presentation (CLIP) numbers, and it is the `Number::Phone::StubCountry` package within that which tripped over the issue.
